source/documentation/runbooks/delayed-job-failures-editor.html.md.erb

```diff
@@ -0,0 +1,37 @@
+---
+title: Editor's Delayed Job Failures
+category: runbooks
+last_reviewed_on: 2023-01-30
+review_in: 3 months
+---
+
+
+<% if current_page.url != "/" %># <%= current_page.data.title %><% end %>
+
+It happens the job could not be processed because the corresponding PublishService id wasn't found.
```

Comment:
Suggest adding a line something like 'in this case there is no issue, we just need to clean up the failed Delayed Job', based on https://github.com/ministryofjustice/moj-forms-tech-docs/pull/44#issuecomment-1410088233